source/gameengine/Launcher/LA_Launcher.cpp

```diff
@@ -276,6 +280,7 @@ void LA_Launcher::InitEngine()
 
 void LA_Launcher::ExitEngine()
 {
+	SCA_Joystick::Close();
```

Comment:
It assumes that the joystick sensor destructor doesn't call any functions in SCA_Joystick.